.github/workflows/operator.yml

```
@@ -0,0 +1,146 @@
+            output:rbac:artifacts:config=config/rbac
+          git diff --exit-code -- api config
+
+  build:
+    needs: test
+    runs-on: ubuntu-latest
+    # Pull requests build but do not publish, so a broken Dockerfile fails in
+    # review rather than on merge, without putting an unreviewed image behind
+    # a tag the cluster might pull.
+    permissions:
+      contents: read
+      packages: ${{ github.event_name == 'pull_request' && 'read' || 'write' }}
```

actionlint (1.7.12)

[error] "${{ github.event_name == 'pull_request' && 'read' || 'write' }}" is invalid as permission of scope "packages". available values are "read", "write", "none"

(permissions)